### PR TITLE
storage: log suspicious none

### PR DESCRIPTION
**Pull request by BusyJay**

### CODE REVIEW

**Comment by BusyJay** [comment]
Enrich the panic message when encounter unexpected None to help debugging.

@disksing @zhangjinpeng1987 PTAL

**Comment by zhangjinpeng1987** [comment]
LGTM

File: src/storage/mvcc/reader.rs [hunk 1]
```diff
@@ -54,9 +54,15 @@ impl<'a> MvccReader<'a> {
 
         let k = key.append_ts(ts);
         if let Some(ref mut cursor) = self.data_cursor {
-            cursor.get(&k).map(|x| x.unwrap().to_vec()).map_err(Error::from)
+            match try!(cursor.get(&k)) {
+                None => panic!("data of [key: {}, ts {}] is missing!", key, ts),
```
Comment (ngaut) [inline src/storage/mvcc/reader.rs]: keys {} not found, ts {}

**Comment by ngaut** [comment]
LGTM

**Comment by disksing** [comment]
LGTM, please update the PR's title.

**Pull request merged**